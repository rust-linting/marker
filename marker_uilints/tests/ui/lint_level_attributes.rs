#![feature(register_tool)]
#![register_tool(marker)]

const FIND_ME_DEFAULT: i32 = 0;

#[allow(marker::item_with_test_name)]
const FIND_ME_ALLOW: i32 = 0;

#[deny(marker::item_with_test_name)]
const FIND_ME_DENY: i32 = 0;
