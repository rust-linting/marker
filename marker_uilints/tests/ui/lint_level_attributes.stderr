warning: found a `const` item with a test name
 --> $DIR/lint_level_attributes.rs:4:1
  |
4 | const FIND_ME_DEFAULT: i32 = 0;
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(marker::item_with_test_name)]` on by default

error: found a `const` item with a test name
  --> $DIR/lint_level_attributes.rs:10:1
   |
10 | const FIND_ME_DENY: i32 = 0;
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
note: the lint level is defined here
  --> $DIR/lint_level_attributes.rs:9:8
   |
9  | #[deny(marker::item_with_test_name)]
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error; 1 warning emitted

